datafusion/core/src/physical_optimizer/enforce_distribution.rs

```diff
@@ -1886,8 +1887,8 @@ pub(crate) mod tests {
 
         // Join on (a == b1)
         let join_on = vec![(
-            Column::new_with_schema("a", &schema()).unwrap(),
-            Column::new_with_schema("b1", &right.schema()).unwrap(),
+            Arc::new(Column::new_with_schema("a", &schema()).unwrap()) as _,
```

Comment:
I think we can use `col` function (which is in the  file`datafusion/physical_expr/src/expressions/column.rs`) here
as `col("a", &schema()).unwrap()` instead of `Arc::new(Column::new_with_schema("a", &schema()).unwrap()) as _`. However, there are lots of changes like this. And it is purely stylistic. This change is not important.